sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/PartitionReceiverLiveTests.cs

```diff
@@ -866,7 +866,7 @@ await using (EventHubScope scope = await EventHubScope.CreateAsync(1))
                 cancellationSource.CancelAfter(EventHubsTestEnvironment.Instance.TestExecutionTimeLimit);
 
                 var connectionString = EventHubsTestEnvironment.Instance.BuildConnectionStringForEventHub(scope.EventHubName);
-                var sourceEvents = EventGenerator.CreateEvents(25).ToList();
+                var sourceEvents = EventGenerator.CreateEvents(250).ToList();
```

Comment:
just curious - why did these increase by 10x and the one above only by 2x?

Reply:
Funny story.  The underlying issue was that the 25 just so happened to align with the prefetch size.   We need there to be several receive attempts to ensure the behavior, and the timing for the close depends on how quickly the service responds.   In most cases, this passed because we had events flowing quickly enough that we had 2-3 receive calls.   In live runs, there were times that execution was paused long enough that prefetch filled up and we only had a single receive.

I bumped this up conservatively to ensure that we had 10 receive calls take place in the worst case, to combat the prefetch filling up and potential slowness of the service to acknowledge and kill the link.   The other location needed only a two receive calls, due to the synchronization, so I gave it a conservative 4 calls.